### PR TITLE
fix: scroll to top on click event

### PR DESCRIPTION
**Pull request by haijian-vaadin**
Apply @abdonrd 's [patch](https://github.com/IBM/pwa-lit-template/blob/main/patches/@vaadin+router+1.7.2.patch) to https://github.com/vaadin/vaadin-router/issues/43. 
Not optimal, but maybe a good first step.

### CODE REVIEW

File: src/triggers/click.js [hunk 1]
```diff
@@ -85,6 +85,11 @@ function vaadinRouterGlobalClickHandler(event) {
     return;
   }
 
+  // for a click event, the scroll is reset to the top position.
+  if (event && event.type === 'click') { 
+    window.scrollTo(0, 0);
```
Comment (platosha) [inline src/triggers/click.js:90]: Maybe we need to move that inside the `if` statement below: only scroll if the link click is handled in the router, i. e., is not outside the router’s base URL.

**Review by platosha — CHANGES_REQUESTED** [review]
Please add a test for this.

**Review by platosha — APPROVED** [review]

**Comment by abdonrd** [comment]
@haijian-vaadin Thanks! Is there an estimate for the next release?

**Comment by haijian-vaadin** [comment]
> @haijian-vaadin Thanks! Is there an estimate for the next release?

Hi @abdonrd, I will try to cherry-pick this fix to 1.x and make a release next Friday (12.2.2012), maybe also make a 2.0 release as well. 
And thanks a lot for your contributions to `vaadin-rouer` 🙇‍♂️.

**Pull request merged**